metadata-ingestion/docs/sources/looker/lookml_recipe.yml

```diff
@@ -29,7 +29,8 @@ source:
     #     platform_instance: bq_warehouse # optional
     #     platform_env: DEV  # optional
 
+    # Optional additional github information. Used to supply github urls to the properties of lookml views.
```

Comment:
Still unsure what "supply github  urls to the properties of lookml views" means if I'm new to DH. No concrete suggestions though

Reply:
made a small tweak!